6. Database/utils/database.js

```diff
@@ -0,0 +1,10 @@
+const mysql = require('mysql2');
+
+const pool = mysql.createPool({
+    host:'localhost',
+    user:'root',
+    database:'Test',
+    password:'Simform@123'
+});
+
```

Comment:
never hard code database or any kind of credentials or kind of secret or private key file in your code. those things must not be committed. always use it from the env file and the env file also must not be committed.

Reply:
yeah, this was just for practice purpose , so i've not did that, but i'll keep that in mind, whenever next time i'll perform any database related stuff.